src/main/java/org/opensearch/security/dlic/rest/api/TenantsApiAction.java

```diff
@@ -108,6 +108,7 @@ public Settings settings() {
                     public Map<String, RequestContentValidator.DataType> allowedKeys() {
                         final ImmutableMap.Builder<String, DataType> allowedKeys = ImmutableMap.builder();
                         if (isCurrentUserAdmin()) {
+                            allowedKeys.put("hidden", DataType.BOOLEAN);
```

Comment:
I [do not see a UT file](https://github.com/opensearch-project/security/tree/main/src/test/java/org/opensearch/security/dlic/rest/api) for this Class, should the same be added with this PR?

Reply:
It is a new test. We never had UI for it (which are Integration Tests in case of REST API)

Reply:
Well, `TenantsV7` extends `Hideable` and `StaticDefinable`. The code was added in 2019. So not clear do we need it or not.

Reply:
IMO we don't use it but I'm not 100% sure. But, if we are already extending Hideable then I think it is fine to add this as an allowed key

Reply:
Same, but it was done in the past :-) and if i remove it it will break backward compatibility